core/node/node_framework/src/implementations/layers/state_keeper/mempool_io.rs

```diff
@@ -86,8 +86,10 @@ impl WiringLayer for MempoolIOLayer {
                 .get_singleton()
                 .await
                 .context("Get master pool")?,
-            self.contracts_config.l2_erc20_bridge_addr,
-            self.state_keeper_config.miniblock_seal_queue_capacity,
+            self.contracts_config
+                .l2_shared_bridge_addr
```

Comment:
cc @kelemeno do we even need two variables? they will always have the same address

Reply:
We need the l2_erc_20_bridge_add for the old SDK, but this could be served using the l2_shared_bridge. 

I think it is for symmetry reasons, so that each bridge has an L1 and L2 component: 

https://github.com/matter-labs/zksync-era/blob/c8dff19deb728e67a9430de8b0f833b1f1da7153/core/lib/protobuf_config/src/proto/config/contracts.proto#L20